ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/api/v1/event/EventOps.scala

```diff
@@ -25,9 +25,14 @@ object EventOps {
       event.copy(event = event.event.modifyWitnessParties(f))
 
     def contractId: String = event.event.contractId
+    def updateContractId(set: String): Event =
```

Comment:
Considering that this is only used for testing, I'd prefer that this helper method and the similar one below is moved to the test class.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/4781/commits/d3e2d32195c2aa697e66260faa86735a0cdb375b